packages/venia-concept/src/RootComponents/CMS/CMS.js

```diff
@@ -1,7 +1,13 @@
 import { createElement, Component } from 'react';
+import Page from 'src/components/Page';
+import CategoryList from 'src/components/CategoryList';
 
 export default class CMS extends Component {
     render() {
-        return <div>CMS Page Stub</div>;
+        return (
+            <Page>
+                <CategoryList title={'Shop by category'} id={2} />
```

Comment:
This is a small thing, but JSX allows you to use literal strings as attribute values without the expression braces. You have to use double quotes, so this could be:
```jsx
<CategoryList title="Shop by category" id={2} />
```
Soon we will create an internationalization system that finds and replaces these strings, so they should use a common syntax.